src/apps/Organization/Organization.js

```diff
@@ -0,0 +1,269 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Box,
+  Button,
+  ButtonText,
+  Header,
+  IconCoin,
+  Info,
+  Layout,
+  GU,
+  textStyle,
+  unselectable,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import LocalIdentityBadge from '../../components/IdentityBadge/LocalIdentityBadge'
+import { appIds, network } from '../../environment'
+import { sanitizeNetworkType } from '../../network-config'
+import { AppType, DaoAddressType, EthereumAddressType } from '../../prop-types'
+import { toChecksumAddress } from '../../web3-utils'
+import airdrop, { testTokensEnabled } from '../../testnet/airdrop'
+import useAppWidth from '../useAppWidth'
+
+const Organization = React.memo(
+  ({
+    account,
+    apps,
+    appsLoading,
+    daoAddress,
+    onOpenApp,
+    walletNetwork,
+    walletWeb3,
+  }) => {
+    const theme = useTheme()
+    const { layoutName } = useLayout()
+
+    const handleDepositTestTokens = useCallback(() => {
+      const finance = apps.find(app => app.appId === appIds.Finance)
+      if (finance && finance.proxyAddress) {
+        airdrop(walletWeb3, finance.proxyAddress, account)
+      }
+    }, [account, apps, walletWeb3])
+    const handleOpenAgentApp = useCallback(() => {
+      const agent = apps.find(app => app.appId === appIds.Agent)
+      if (agent && agent.proxyAddress) {
+        onOpenApp(agent.proxyAddress)
+      }
+    }, [apps, onOpenApp])
+    const handleOpenFinanceApp = useCallback(() => {
+      const finance = apps.find(app => app.appId === appIds.Finance)
+      if (finance && finance.proxyAddress) {
+        onOpenApp(finance.proxyAddress)
+      }
+    }, [apps, onOpenApp])
+
+    const apmApps = apps.filter(app => !app.isAragonOsInternalApp)
+    const hasAgentApp = apps.some(app => app.appId === appIds.Agent)
+    const hasFinanceApp = apps.some(app => app.appId === appIds.Finance)
+    const checksummedDaoAddr =
+      daoAddress.address && toChecksumAddress(daoAddress.address)
+    const enableTransactions = !!account && walletNetwork === network.type
+    const shortAddresses = layoutName !== 'large'
+
+    const depositFundsHelpText = appsLoading ? (
+      ''
+    ) : hasFinanceApp || hasAgentApp ? (
+      <span>
+        If you'd like to deposit funds into this organization, you can do so
+        from{' '}
+        {hasFinanceApp ? (
+          <OpenAppButton onClick={handleOpenFinanceApp}>Finance</OpenAppButton>
+        ) : (
+          <OpenAppButton onClick={handleOpenAgentApp}>Agent</OpenAppButton>
+        )}
+        .
+      </span>
+    ) : (
+      `This organization does not have a Finance or Agent app installed and may
+       not be able to receive funds. Please check with the organization's
```

Comment:
```suggestion
       not be able to receive funds. Please check with the organization’s
```

Reply:
Interesting, is it preferred to use these in text like the double quotes? I've never used them before for copy.

Reply:
Yes `’` is an actual apostrophe, while `'` is a single quote (but extremely often used as an apostrophe due to keyboard layouts :o)